src/unix/fs.c

```diff
@@ -231,7 +231,45 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 
 
 static ssize_t uv__fs_mkdtemp(uv_fs_t* req) {
+#if defined(__MVS__)
+  /* There is no mkdtemp. So instead use mktemp to generate a
```

Comment:
are you sure this is completely functionally equivalent? I suggest looking at fs__mkdtemp from src/win/fs.c, it has a unix equivalent mkdtemp(). You  can't use it as-is, because of the windows specific string handling, but you could probably modify it, or go back to the source, see https://github.com/libuv/libuv/blob/531f06ecfe603517391340e585597bbc91b4a66b/src/win/fs.c#L765-L766, and get the MIT licensed unixy code.